DataDictionaries/Epi25GGE.csv

```diff
@@ -83,4 +87,6 @@ family_history,clinical,,dropdown,Family History,"1, Yes | 2, No | 998, Unknown"

```

Comment:
syndrome_focal was introduced in 5b57b230d68b6105e5d69a79dbb2b075e863fd84
Which one is more preferable?